src/python/pants/pantsd/pants_daemon.py

```diff
@@ -169,6 +169,20 @@ def watchman_launcher(self):
   def is_killed(self):
     return self._kill_switch.is_set()
 
+  @property
+  def options_fingerprint(self):
+    return self._bootstrap_options.sha1(
```

Comment:
There is already a facility for marking options as `fingerprint=True`... it would be good to reuse that here if possible: https://github.com/pantsbuild/pants/blob/33e98e1869c64ad277130807afe989988e83063c/src/python/pants/option/options.py#L320

Reply:
great idea. refactored.